openssl-dynamic/src/main/c/ssl.c

```diff
@@ -1666,30 +1560,26 @@ TCN_IMPLEMENT_CALL(void, SSL, setState)(TCN_STDARGS,
                                            jint state) {
     SSL *ssl_ = J2P(ssl, SSL *);
 
-    if (ssl_ == NULL) {
-        tcn_ThrowException(e, "ssl is null");
-        return;
-    }
+    TCN_CHECK_NULL(ssl_, "ssl is null",);
```

Comment:
consider clarifying this as follow: `TCN_CHECK_NULL(ssl_, "ssl is null", /*void*/);`